FrontEnd/src/components/AboutPage/MeetTheTeam/MeetTheTeam.tsx

```diff
@@ -1,7 +1,126 @@
 import React, { FunctionComponent } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
 import styles from './MeetTheTeam.module.scss';
+import MemberInfoCard, {
+  MemberInfoCardProps,
+} from './MemberInfoCard/MemberInfoCard';
 
-const MeetTheTeam: FunctionComponent = () => <Container></Container>;
+const Cards: MemberInfoCardProps[] = [
```

Comment:
It might be a good idea to have the card info array exported from a different file and then used here. Also, you should add "as const" at the end to have typescript enforce no changes to the variable (see here: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html)